packages/account/src/Components/poi/poi-button/__tests__/poi-button.spec.tsx

```diff
@@ -18,7 +18,8 @@ describe('<PoiButton/>', () => {
     it('should navigate to proof_of_identity page after clicking', () => {
         renderWithRouter(<PoiButton />);
         const poi_button = screen.getByText('Proof of identity');
-        expect(poi_button.closest('a')).toHaveAttribute('href', '/account/proof-of-identity');
+        const link = screen.getByRole('link', { current: false });
```

Comment:
what does this current:false do?

Reply:
It picks the element whose aria-current is set to false. Here I am using it for picking the link element